[2024-02-27T12:07:12.656+0700] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: missed_deadline_job_dag.spark_ingestion_to_s3 manual__2024-02-26T09:28:46.151976+00:00 [queued]>
[2024-02-27T12:07:12.703+0700] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: missed_deadline_job_dag.spark_ingestion_to_s3 manual__2024-02-26T09:28:46.151976+00:00 [queued]>
[2024-02-27T12:07:12.703+0700] {taskinstance.py:2170} INFO - Starting attempt 5 of 5
[2024-02-27T12:07:12.757+0700] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): spark_ingestion_to_s3> on 2024-02-26 09:28:46.151976+00:00
[2024-02-27T12:07:12.761+0700] {standard_task_runner.py:60} INFO - Started process 32232 to run task
[2024-02-27T12:07:12.771+0700] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'missed_deadline_job_dag', 'spark_ingestion_to_s3', 'manual__2024-02-26T09:28:46.151976+00:00', '--job-id', '153', '--raw', '--subdir', 'DAGS_FOLDER/missed_deadlinee_dag.py', '--cfg-path', '/tmp/tmpfpc8wjuv']
[2024-02-27T12:07:12.775+0700] {standard_task_runner.py:88} INFO - Job 153: Subtask spark_ingestion_to_s3
[2024-02-27T12:07:13.022+0700] {task_command.py:423} INFO - Running <TaskInstance: missed_deadline_job_dag.spark_ingestion_to_s3 manual__2024-02-26T09:28:46.151976+00:00 [running]> on host MinhNhut.
[2024-02-27T12:07:13.398+0700] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='missed_deadline_job_dag' AIRFLOW_CTX_TASK_ID='spark_ingestion_to_s3' AIRFLOW_CTX_EXECUTION_DATE='2024-02-26T09:28:46.151976+00:00' AIRFLOW_CTX_TRY_NUMBER='5' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-02-26T09:28:46.151976+00:00'
[2024-02-27T12:07:13.402+0700] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-02-27T12:07:13.403+0700] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python /home/nhut/etl-***/pipeline/spark_ingestion_to_s3.py']
[2024-02-27T12:07:13.441+0700] {subprocess.py:86} INFO - Output:
[2024-02-27T12:07:23.846+0700] {subprocess.py:93} INFO - 24/02/27 12:07:23 WARN Utils: Your hostname, MinhNhut resolves to a loopback address: 127.0.1.1; using 172.28.89.67 instead (on interface eth0)
[2024-02-27T12:07:23.856+0700] {subprocess.py:93} INFO - 24/02/27 12:07:23 WARN Utils: Set SPARK_LOCAL_IP if you need to bind to another address
[2024-02-27T12:07:34.992+0700] {subprocess.py:93} INFO - :: loading settings :: url = jar:file:/home/nhut/etl-***/.venv/lib/python3.10/site-packages/pyspark/jars/ivy-2.5.1.jar!/org/apache/ivy/core/settings/ivysettings.xml
[2024-02-27T12:07:35.585+0700] {subprocess.py:93} INFO - Ivy Default Cache set to: /home/nhut/.ivy2/cache
[2024-02-27T12:07:35.586+0700] {subprocess.py:93} INFO - The jars for the packages stored in: /home/nhut/.ivy2/jars
[2024-02-27T12:07:35.611+0700] {subprocess.py:93} INFO - org.apache.hadoop#hadoop-aws added as a dependency
[2024-02-27T12:07:35.618+0700] {subprocess.py:93} INFO - :: resolving dependencies :: org.apache.spark#spark-submit-parent-1f34ba63-a21b-4fcb-a068-be9d56f062b4;1.0
[2024-02-27T12:07:35.619+0700] {subprocess.py:93} INFO - 	confs: [default]
[2024-02-27T12:07:36.622+0700] {subprocess.py:93} INFO - 	found org.apache.hadoop#hadoop-aws;3.3.2 in spark-list
[2024-02-27T12:07:36.794+0700] {subprocess.py:93} INFO - 	found com.amazonaws#aws-java-sdk-bundle;1.11.1026 in spark-list
[2024-02-27T12:07:36.989+0700] {subprocess.py:93} INFO - 	found org.wildfly.openssl#wildfly-openssl;1.0.7.Final in central
[2024-02-27T12:07:37.122+0700] {subprocess.py:93} INFO - :: resolution report :: resolve 1456ms :: artifacts dl 41ms
[2024-02-27T12:07:37.123+0700] {subprocess.py:93} INFO - 	:: modules in use:
[2024-02-27T12:07:37.124+0700] {subprocess.py:93} INFO - 	com.amazonaws#aws-java-sdk-bundle;1.11.1026 from spark-list in [default]
[2024-02-27T12:07:37.124+0700] {subprocess.py:93} INFO - 	org.apache.hadoop#hadoop-aws;3.3.2 from spark-list in [default]
[2024-02-27T12:07:37.125+0700] {subprocess.py:93} INFO - 	org.wildfly.openssl#wildfly-openssl;1.0.7.Final from central in [default]
[2024-02-27T12:07:37.125+0700] {subprocess.py:93} INFO - 	---------------------------------------------------------------------
[2024-02-27T12:07:37.126+0700] {subprocess.py:93} INFO - 	|                  |            modules            ||   artifacts   |
[2024-02-27T12:07:37.126+0700] {subprocess.py:93} INFO - 	|       conf       | number| search|dwnlded|evicted|| number|dwnlded|
[2024-02-27T12:07:37.127+0700] {subprocess.py:93} INFO - 	---------------------------------------------------------------------
[2024-02-27T12:07:37.127+0700] {subprocess.py:93} INFO - 	|      default     |   3   |   0   |   0   |   0   ||   3   |   0   |
[2024-02-27T12:07:37.128+0700] {subprocess.py:93} INFO - 	---------------------------------------------------------------------
[2024-02-27T12:07:37.162+0700] {subprocess.py:93} INFO - :: retrieving :: org.apache.spark#spark-submit-parent-1f34ba63-a21b-4fcb-a068-be9d56f062b4
[2024-02-27T12:07:37.163+0700] {subprocess.py:93} INFO - 	confs: [default]
[2024-02-27T12:07:37.192+0700] {subprocess.py:93} INFO - 	0 artifacts copied, 3 already retrieved (0kB/13ms)
[2024-02-27T12:07:38.322+0700] {subprocess.py:93} INFO - 24/02/27 12:07:38 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2024-02-27T12:07:39.512+0700] {subprocess.py:93} INFO - Setting default log level to "WARN".
[2024-02-27T12:07:39.513+0700] {subprocess.py:93} INFO - To adjust logging level use sc.setLogLevel(newLevel). For SparkR, use setLogLevel(newLevel).
[2024-02-27T12:07:45.616+0700] {subprocess.py:93} INFO - 24/02/27 12:07:45 WARN Utils: Service 'SparkUI' could not bind on port 4040. Attempting port 4041.
[2024-02-27T12:07:45.619+0700] {subprocess.py:93} INFO - 24/02/27 12:07:45 WARN Utils: Service 'SparkUI' could not bind on port 4041. Attempting port 4042.
[2024-02-27T12:07:45.625+0700] {subprocess.py:93} INFO - 24/02/27 12:07:45 WARN Utils: Service 'SparkUI' could not bind on port 4042. Attempting port 4043.
[2024-02-27T12:07:45.627+0700] {subprocess.py:93} INFO - 24/02/27 12:07:45 WARN Utils: Service 'SparkUI' could not bind on port 4043. Attempting port 4044.
[2024-02-27T12:07:45.635+0700] {subprocess.py:93} INFO - 24/02/27 12:07:45 WARN Utils: Service 'SparkUI' could not bind on port 4044. Attempting port 4045.
[2024-02-27T12:07:45.636+0700] {subprocess.py:93} INFO - 24/02/27 12:07:45 WARN Utils: Service 'SparkUI' could not bind on port 4045. Attempting port 4046.
[2024-02-27T12:07:45.640+0700] {subprocess.py:93} INFO - 24/02/27 12:07:45 WARN Utils: Service 'SparkUI' could not bind on port 4046. Attempting port 4047.
[2024-02-27T12:07:45.649+0700] {subprocess.py:93} INFO - 24/02/27 12:07:45 WARN Utils: Service 'SparkUI' could not bind on port 4047. Attempting port 4048.
[2024-02-27T12:12:38.343+0700] {subprocess.py:93} INFO - [Stage 0:>                                                          (0 + 0) / 1][Stage 0:>                                                          (0 + 1) / 1][Stage 0:===========================================================(1 + 0) / 1]                                                                                24/02/27 12:12:38 WARN MetricsConfig: Cannot locate configuration: tried hadoop-metrics2-s3a-file-system.properties,hadoop-metrics2.properties
[2024-02-27T12:14:17.942+0700] {subprocess.py:93} INFO - [Stage 1:>                                                          (0 + 4) / 4][Stage 1:==============>                                            (1 + 3) / 4][Stage 1:============================================>              (3 + 1) / 4]                                                                                [Stage 3:>                                                          (0 + 2) / 2][Stage 3:=============================>                             (1 + 1) / 2]                                                                                24/02/27 12:14:17 ERROR FileFormatWriter: Aborting job 2e147cb4-0523-42bf-b136-c4108609fffd.
[2024-02-27T12:14:17.976+0700] {subprocess.py:93} INFO - org.apache.hadoop.fs.s3a.RemoteFileChangedException: copyFile(olist_order_payments_dataset.parquet/_temporary/0/task_202402271213478392135236523862282_0003_m_000000/part-00000-a686593a-e86f-48d6-9467-e8b024a46016-c000.snappy.parquet, olist_order_payments_dataset.parquet/part-00000-a686593a-e86f-48d6-9467-e8b024a46016-c000.snappy.parquet) `s3a://olistsalesbucket/olist_order_payments_dataset.parquet/_temporary/0/task_202402271213478392135236523862282_0003_m_000000/part-00000-a686593a-e86f-48d6-9467-e8b024a46016-c000.snappy.parquet': File to rename not found on unguarded S3 store: copyFile(olist_order_payments_dataset.parquet/_temporary/0/task_202402271213478392135236523862282_0003_m_000000/part-00000-a686593a-e86f-48d6-9467-e8b024a46016-c000.snappy.parquet, olist_order_payments_dataset.parquet/part-00000-a686593a-e86f-48d6-9467-e8b024a46016-c000.snappy.parquet) on olist_order_payments_dataset.parquet/_temporary/0/task_202402271213478392135236523862282_0003_m_000000/part-00000-a686593a-e86f-48d6-9467-e8b024a46016-c000.snappy.parquet: com.amazonaws.services.s3.model.AmazonS3Exception: Not Found (Service: Amazon S3; Status Code: 404; Error Code: 404 Not Found; Request ID: NNN41GP1M7MJMW0H; S3 Extended Request ID: nUeNRiQgZyOyrOwd6Dp/3G74ywjtdmwXt/O6TCBvdaMRDgxM9fY40421GF+mawJtIHjHgCb0gz0=; Proxy: null), S3 Extended Request ID: nUeNRiQgZyOyrOwd6Dp/3G74ywjtdmwXt/O6TCBvdaMRDgxM9fY40421GF+mawJtIHjHgCb0gz0=:404 Not Found
[2024-02-27T12:14:17.976+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.copyFile(S3AFileSystem.java:4302)
[2024-02-27T12:14:17.977+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.access$1200(S3AFileSystem.java:259)
[2024-02-27T12:14:17.977+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem$OperationCallbacksImpl.copyFile(S3AFileSystem.java:2084)
[2024-02-27T12:14:17.977+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.impl.RenameOperation.copySourceAndUpdateTracker(RenameOperation.java:638)
[2024-02-27T12:14:17.977+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.impl.RenameOperation.renameFileToDest(RenameOperation.java:365)
[2024-02-27T12:14:17.977+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.impl.RenameOperation.execute(RenameOperation.java:310)
[2024-02-27T12:14:17.977+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.innerRename(S3AFileSystem.java:1999)
[2024-02-27T12:14:17.978+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.lambda$rename$7(S3AFileSystem.java:1846)
[2024-02-27T12:14:17.978+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.statistics.impl.IOStatisticsBinding.lambda$trackDurationOfOperation$5(IOStatisticsBinding.java:499)
[2024-02-27T12:14:17.978+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.statistics.impl.IOStatisticsBinding.trackDuration(IOStatisticsBinding.java:444)
[2024-02-27T12:14:17.978+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.trackDurationAndSpan(S3AFileSystem.java:2337)
[2024-02-27T12:14:17.978+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.rename(S3AFileSystem.java:1844)
[2024-02-27T12:14:17.978+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter.mergePaths(FileOutputCommitter.java:476)
[2024-02-27T12:14:17.978+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter.mergePaths(FileOutputCommitter.java:490)
[2024-02-27T12:14:17.978+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter.commitJobInternal(FileOutputCommitter.java:405)
[2024-02-27T12:14:17.978+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter.commitJob(FileOutputCommitter.java:377)
[2024-02-27T12:14:17.978+0700] {subprocess.py:93} INFO - 	at org.apache.parquet.hadoop.ParquetOutputCommitter.commitJob(ParquetOutputCommitter.java:48)
[2024-02-27T12:14:17.978+0700] {subprocess.py:93} INFO - 	at org.apache.spark.internal.io.HadoopMapReduceCommitProtocol.commitJob(HadoopMapReduceCommitProtocol.scala:192)
[2024-02-27T12:14:17.979+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.$anonfun$writeAndCommit$3(FileFormatWriter.scala:275)
[2024-02-27T12:14:17.979+0700] {subprocess.py:93} INFO - 	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
[2024-02-27T12:14:17.979+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.timeTakenMs(Utils.scala:552)
[2024-02-27T12:14:17.980+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.writeAndCommit(FileFormatWriter.scala:275)
[2024-02-27T12:14:17.980+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.executeWrite(FileFormatWriter.scala:304)
[2024-02-27T12:14:17.980+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.write(FileFormatWriter.scala:190)
[2024-02-27T12:14:17.980+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.InsertIntoHadoopFsRelationCommand.run(InsertIntoHadoopFsRelationCommand.scala:190)
[2024-02-27T12:14:17.980+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.command.DataWritingCommandExec.sideEffectResult$lzycompute(commands.scala:113)
[2024-02-27T12:14:17.980+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.command.DataWritingCommandExec.sideEffectResult(commands.scala:111)
[2024-02-27T12:14:17.980+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.command.DataWritingCommandExec.executeCollect(commands.scala:125)
[2024-02-27T12:14:17.980+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.QueryExecution$$anonfun$eagerlyExecuteCommands$1.$anonfun$applyOrElse$1(QueryExecution.scala:107)
[2024-02-27T12:14:17.980+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SQLExecution$.$anonfun$withNewExecutionId$6(SQLExecution.scala:125)
[2024-02-27T12:14:17.980+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SQLExecution$.withSQLConfPropagated(SQLExecution.scala:201)
[2024-02-27T12:14:17.980+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SQLExecution$.$anonfun$withNewExecutionId$1(SQLExecution.scala:108)
[2024-02-27T12:14:17.981+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.SparkSession.withActive(SparkSession.scala:900)
[2024-02-27T12:14:17.981+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SQLExecution$.withNewExecutionId(SQLExecution.scala:66)
[2024-02-27T12:14:17.981+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.QueryExecution$$anonfun$eagerlyExecuteCommands$1.applyOrElse(QueryExecution.scala:107)
[2024-02-27T12:14:17.981+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.QueryExecution$$anonfun$eagerlyExecuteCommands$1.applyOrElse(QueryExecution.scala:98)
[2024-02-27T12:14:17.981+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.trees.TreeNode.$anonfun$transformDownWithPruning$1(TreeNode.scala:461)
[2024-02-27T12:14:17.981+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.trees.CurrentOrigin$.withOrigin(origin.scala:76)
[2024-02-27T12:14:17.981+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.trees.TreeNode.transformDownWithPruning(TreeNode.scala:461)
[2024-02-27T12:14:17.981+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.plans.logical.LogicalPlan.org$apache$spark$sql$catalyst$plans$logical$AnalysisHelper$$super$transformDownWithPruning(LogicalPlan.scala:32)
[2024-02-27T12:14:17.982+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.plans.logical.AnalysisHelper.transformDownWithPruning(AnalysisHelper.scala:267)
[2024-02-27T12:14:17.982+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.plans.logical.AnalysisHelper.transformDownWithPruning$(AnalysisHelper.scala:263)
[2024-02-27T12:14:17.982+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.plans.logical.LogicalPlan.transformDownWithPruning(LogicalPlan.scala:32)
[2024-02-27T12:14:17.982+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.plans.logical.LogicalPlan.transformDownWithPruning(LogicalPlan.scala:32)
[2024-02-27T12:14:17.982+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.trees.TreeNode.transformDown(TreeNode.scala:437)
[2024-02-27T12:14:17.982+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.QueryExecution.eagerlyExecuteCommands(QueryExecution.scala:98)
[2024-02-27T12:14:17.982+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.QueryExecution.commandExecuted$lzycompute(QueryExecution.scala:85)
[2024-02-27T12:14:17.983+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.QueryExecution.commandExecuted(QueryExecution.scala:83)
[2024-02-27T12:14:17.983+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.QueryExecution.assertCommandExecuted(QueryExecution.scala:142)
[2024-02-27T12:14:17.983+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.DataFrameWriter.runCommand(DataFrameWriter.scala:859)
[2024-02-27T12:14:17.983+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.DataFrameWriter.saveToV1Source(DataFrameWriter.scala:388)
[2024-02-27T12:14:17.983+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.DataFrameWriter.saveInternal(DataFrameWriter.scala:361)
[2024-02-27T12:14:17.983+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.DataFrameWriter.save(DataFrameWriter.scala:240)
[2024-02-27T12:14:17.983+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.DataFrameWriter.parquet(DataFrameWriter.scala:792)
[2024-02-27T12:14:17.983+0700] {subprocess.py:93} INFO - 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[2024-02-27T12:14:17.983+0700] {subprocess.py:93} INFO - 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[2024-02-27T12:14:17.984+0700] {subprocess.py:93} INFO - 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[2024-02-27T12:14:17.984+0700] {subprocess.py:93} INFO - 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[2024-02-27T12:14:17.984+0700] {subprocess.py:93} INFO - 	at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:244)
[2024-02-27T12:14:17.984+0700] {subprocess.py:93} INFO - 	at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:374)
[2024-02-27T12:14:17.984+0700] {subprocess.py:93} INFO - 	at py4j.Gateway.invoke(Gateway.java:282)
[2024-02-27T12:14:17.984+0700] {subprocess.py:93} INFO - 	at py4j.commands.AbstractCommand.invokeMethod(AbstractCommand.java:132)
[2024-02-27T12:14:17.984+0700] {subprocess.py:93} INFO - 	at py4j.commands.CallCommand.execute(CallCommand.java:79)
[2024-02-27T12:14:17.984+0700] {subprocess.py:93} INFO - 	at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2024-02-27T12:14:17.984+0700] {subprocess.py:93} INFO - 	at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2024-02-27T12:14:17.984+0700] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:829)
[2024-02-27T12:14:17.985+0700] {subprocess.py:93} INFO - Caused by: java.io.FileNotFoundException: copyFile(olist_order_payments_dataset.parquet/_temporary/0/task_202402271213478392135236523862282_0003_m_000000/part-00000-a686593a-e86f-48d6-9467-e8b024a46016-c000.snappy.parquet, olist_order_payments_dataset.parquet/part-00000-a686593a-e86f-48d6-9467-e8b024a46016-c000.snappy.parquet) on olist_order_payments_dataset.parquet/_temporary/0/task_202402271213478392135236523862282_0003_m_000000/part-00000-a686593a-e86f-48d6-9467-e8b024a46016-c000.snappy.parquet: com.amazonaws.services.s3.model.AmazonS3Exception: Not Found (Service: Amazon S3; Status Code: 404; Error Code: 404 Not Found; Request ID: NNN41GP1M7MJMW0H; S3 Extended Request ID: nUeNRiQgZyOyrOwd6Dp/3G74ywjtdmwXt/O6TCBvdaMRDgxM9fY40421GF+mawJtIHjHgCb0gz0=; Proxy: null), S3 Extended Request ID: nUeNRiQgZyOyrOwd6Dp/3G74ywjtdmwXt/O6TCBvdaMRDgxM9fY40421GF+mawJtIHjHgCb0gz0=:404 Not Found
[2024-02-27T12:14:17.986+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AUtils.translateException(S3AUtils.java:266)
[2024-02-27T12:14:17.986+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.Invoker.once(Invoker.java:119)
[2024-02-27T12:14:17.986+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.copyFile(S3AFileSystem.java:4284)
[2024-02-27T12:14:17.986+0700] {subprocess.py:93} INFO - 	... 65 more
[2024-02-27T12:14:17.987+0700] {subprocess.py:93} INFO - Caused by: com.amazonaws.services.s3.model.AmazonS3Exception: Not Found (Service: Amazon S3; Status Code: 404; Error Code: 404 Not Found; Request ID: NNN41GP1M7MJMW0H; S3 Extended Request ID: nUeNRiQgZyOyrOwd6Dp/3G74ywjtdmwXt/O6TCBvdaMRDgxM9fY40421GF+mawJtIHjHgCb0gz0=; Proxy: null), S3 Extended Request ID: nUeNRiQgZyOyrOwd6Dp/3G74ywjtdmwXt/O6TCBvdaMRDgxM9fY40421GF+mawJtIHjHgCb0gz0=
[2024-02-27T12:14:17.987+0700] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.handleErrorResponse(AmazonHttpClient.java:1819)
[2024-02-27T12:14:17.987+0700] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.handleServiceErrorResponse(AmazonHttpClient.java:1403)
[2024-02-27T12:14:17.987+0700] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeOneRequest(AmazonHttpClient.java:1372)
[2024-02-27T12:14:17.987+0700] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeHelper(AmazonHttpClient.java:1145)
[2024-02-27T12:14:17.987+0700] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.doExecute(AmazonHttpClient.java:802)
[2024-02-27T12:14:17.987+0700] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeWithTimer(AmazonHttpClient.java:770)
[2024-02-27T12:14:17.987+0700] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.execute(AmazonHttpClient.java:744)
[2024-02-27T12:14:17.987+0700] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.access$500(AmazonHttpClient.java:704)
[2024-02-27T12:14:17.988+0700] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutionBuilderImpl.execute(AmazonHttpClient.java:686)
[2024-02-27T12:14:17.988+0700] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient.execute(AmazonHttpClient.java:550)
[2024-02-27T12:14:17.988+0700] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient.execute(AmazonHttpClient.java:530)
[2024-02-27T12:14:17.988+0700] {subprocess.py:93} INFO - 	at com.amazonaws.services.s3.AmazonS3Client.invoke(AmazonS3Client.java:5437)
[2024-02-27T12:14:17.988+0700] {subprocess.py:93} INFO - 	at com.amazonaws.services.s3.AmazonS3Client.invoke(AmazonS3Client.java:5384)
[2024-02-27T12:14:17.988+0700] {subprocess.py:93} INFO - 	at com.amazonaws.services.s3.AmazonS3Client.getObjectMetadata(AmazonS3Client.java:1367)
[2024-02-27T12:14:17.988+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.lambda$getObjectMetadata$10(S3AFileSystem.java:2545)
[2024-02-27T12:14:17.988+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.Invoker.retryUntranslated(Invoker.java:414)
[2024-02-27T12:14:17.988+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.Invoker.retryUntranslated(Invoker.java:377)
[2024-02-27T12:14:17.988+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.getObjectMetadata(S3AFileSystem.java:2533)
[2024-02-27T12:14:17.988+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.lambda$copyFile$28(S3AFileSystem.java:4286)
[2024-02-27T12:14:17.989+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.Invoker.once(Invoker.java:117)
[2024-02-27T12:14:17.989+0700] {subprocess.py:93} INFO - 	... 66 more
[2024-02-27T12:14:23.137+0700] {subprocess.py:93} INFO - Traceback (most recent call last):
[2024-02-27T12:14:23.138+0700] {subprocess.py:93} INFO -   File "/home/nhut/etl-***/pipeline/spark_ingestion_to_s3.py", line 31, in <module>
[2024-02-27T12:14:23.144+0700] {subprocess.py:93} INFO -     extract_data()
[2024-02-27T12:14:23.145+0700] {subprocess.py:93} INFO -   File "/home/nhut/etl-***/pipeline/spark_ingestion_to_s3.py", line 27, in extract_data
[2024-02-27T12:14:23.145+0700] {subprocess.py:93} INFO -     df.write.mode("overwrite").parquet("s3a://olistsalesbucket/" + name_file + ".parquet")
[2024-02-27T12:14:23.145+0700] {subprocess.py:93} INFO -   File "/home/nhut/etl-***/.venv/lib/python3.10/site-packages/pyspark/sql/readwriter.py", line 1721, in parquet
[2024-02-27T12:14:23.145+0700] {subprocess.py:93} INFO -     self._jwrite.parquet(path)
[2024-02-27T12:14:23.145+0700] {subprocess.py:93} INFO -   File "/home/nhut/etl-***/.venv/lib/python3.10/site-packages/py4j/java_gateway.py", line 1322, in __call__
[2024-02-27T12:14:23.147+0700] {subprocess.py:93} INFO -     return_value = get_return_value(
[2024-02-27T12:14:23.147+0700] {subprocess.py:93} INFO -   File "/home/nhut/etl-***/.venv/lib/python3.10/site-packages/pyspark/errors/exceptions/captured.py", line 179, in deco
[2024-02-27T12:14:23.147+0700] {subprocess.py:93} INFO -     return f(*a, **kw)
[2024-02-27T12:14:23.149+0700] {subprocess.py:93} INFO -   File "/home/nhut/etl-***/.venv/lib/python3.10/site-packages/py4j/protocol.py", line 326, in get_return_value
[2024-02-27T12:14:23.149+0700] {subprocess.py:93} INFO -     raise Py4JJavaError(
[2024-02-27T12:14:23.183+0700] {subprocess.py:93} INFO - py4j.protocol.Py4JJavaError: An error occurred while calling o42.parquet.
[2024-02-27T12:14:23.183+0700] {subprocess.py:93} INFO - : org.apache.hadoop.fs.s3a.RemoteFileChangedException: copyFile(olist_order_payments_dataset.parquet/_temporary/0/task_202402271213478392135236523862282_0003_m_000000/part-00000-a686593a-e86f-48d6-9467-e8b024a46016-c000.snappy.parquet, olist_order_payments_dataset.parquet/part-00000-a686593a-e86f-48d6-9467-e8b024a46016-c000.snappy.parquet) `s3a://olistsalesbucket/olist_order_payments_dataset.parquet/_temporary/0/task_202402271213478392135236523862282_0003_m_000000/part-00000-a686593a-e86f-48d6-9467-e8b024a46016-c000.snappy.parquet': File to rename not found on unguarded S3 store: copyFile(olist_order_payments_dataset.parquet/_temporary/0/task_202402271213478392135236523862282_0003_m_000000/part-00000-a686593a-e86f-48d6-9467-e8b024a46016-c000.snappy.parquet, olist_order_payments_dataset.parquet/part-00000-a686593a-e86f-48d6-9467-e8b024a46016-c000.snappy.parquet) on olist_order_payments_dataset.parquet/_temporary/0/task_202402271213478392135236523862282_0003_m_000000/part-00000-a686593a-e86f-48d6-9467-e8b024a46016-c000.snappy.parquet: com.amazonaws.services.s3.model.AmazonS3Exception: Not Found (Service: Amazon S3; Status Code: 404; Error Code: 404 Not Found; Request ID: NNN41GP1M7MJMW0H; S3 Extended Request ID: nUeNRiQgZyOyrOwd6Dp/3G74ywjtdmwXt/O6TCBvdaMRDgxM9fY40421GF+mawJtIHjHgCb0gz0=; Proxy: null), S3 Extended Request ID: nUeNRiQgZyOyrOwd6Dp/3G74ywjtdmwXt/O6TCBvdaMRDgxM9fY40421GF+mawJtIHjHgCb0gz0=:404 Not Found
[2024-02-27T12:14:23.183+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.copyFile(S3AFileSystem.java:4302)
[2024-02-27T12:14:23.183+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.access$1200(S3AFileSystem.java:259)
[2024-02-27T12:14:23.184+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem$OperationCallbacksImpl.copyFile(S3AFileSystem.java:2084)
[2024-02-27T12:14:23.184+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.impl.RenameOperation.copySourceAndUpdateTracker(RenameOperation.java:638)
[2024-02-27T12:14:23.184+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.impl.RenameOperation.renameFileToDest(RenameOperation.java:365)
[2024-02-27T12:14:23.184+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.impl.RenameOperation.execute(RenameOperation.java:310)
[2024-02-27T12:14:23.184+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.innerRename(S3AFileSystem.java:1999)
[2024-02-27T12:14:23.184+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.lambda$rename$7(S3AFileSystem.java:1846)
[2024-02-27T12:14:23.184+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.statistics.impl.IOStatisticsBinding.lambda$trackDurationOfOperation$5(IOStatisticsBinding.java:499)
[2024-02-27T12:14:23.184+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.statistics.impl.IOStatisticsBinding.trackDuration(IOStatisticsBinding.java:444)
[2024-02-27T12:14:23.185+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.trackDurationAndSpan(S3AFileSystem.java:2337)
[2024-02-27T12:14:23.185+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.rename(S3AFileSystem.java:1844)
[2024-02-27T12:14:23.185+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter.mergePaths(FileOutputCommitter.java:476)
[2024-02-27T12:14:23.185+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter.mergePaths(FileOutputCommitter.java:490)
[2024-02-27T12:14:23.185+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter.commitJobInternal(FileOutputCommitter.java:405)
[2024-02-27T12:14:23.185+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter.commitJob(FileOutputCommitter.java:377)
[2024-02-27T12:14:23.185+0700] {subprocess.py:93} INFO - 	at org.apache.parquet.hadoop.ParquetOutputCommitter.commitJob(ParquetOutputCommitter.java:48)
[2024-02-27T12:14:23.185+0700] {subprocess.py:93} INFO - 	at org.apache.spark.internal.io.HadoopMapReduceCommitProtocol.commitJob(HadoopMapReduceCommitProtocol.scala:192)
[2024-02-27T12:14:23.185+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.$anonfun$writeAndCommit$3(FileFormatWriter.scala:275)
[2024-02-27T12:14:23.185+0700] {subprocess.py:93} INFO - 	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
[2024-02-27T12:14:23.186+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.timeTakenMs(Utils.scala:552)
[2024-02-27T12:14:23.186+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.writeAndCommit(FileFormatWriter.scala:275)
[2024-02-27T12:14:23.186+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.executeWrite(FileFormatWriter.scala:304)
[2024-02-27T12:14:23.186+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.write(FileFormatWriter.scala:190)
[2024-02-27T12:14:23.186+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.InsertIntoHadoopFsRelationCommand.run(InsertIntoHadoopFsRelationCommand.scala:190)
[2024-02-27T12:14:23.186+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.command.DataWritingCommandExec.sideEffectResult$lzycompute(commands.scala:113)
[2024-02-27T12:14:23.186+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.command.DataWritingCommandExec.sideEffectResult(commands.scala:111)
[2024-02-27T12:14:23.186+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.command.DataWritingCommandExec.executeCollect(commands.scala:125)
[2024-02-27T12:14:23.186+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.QueryExecution$$anonfun$eagerlyExecuteCommands$1.$anonfun$applyOrElse$1(QueryExecution.scala:107)
[2024-02-27T12:14:23.187+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SQLExecution$.$anonfun$withNewExecutionId$6(SQLExecution.scala:125)
[2024-02-27T12:14:23.187+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SQLExecution$.withSQLConfPropagated(SQLExecution.scala:201)
[2024-02-27T12:14:23.187+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SQLExecution$.$anonfun$withNewExecutionId$1(SQLExecution.scala:108)
[2024-02-27T12:14:23.187+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.SparkSession.withActive(SparkSession.scala:900)
[2024-02-27T12:14:23.187+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SQLExecution$.withNewExecutionId(SQLExecution.scala:66)
[2024-02-27T12:14:23.187+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.QueryExecution$$anonfun$eagerlyExecuteCommands$1.applyOrElse(QueryExecution.scala:107)
[2024-02-27T12:14:23.187+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.QueryExecution$$anonfun$eagerlyExecuteCommands$1.applyOrElse(QueryExecution.scala:98)
[2024-02-27T12:14:23.187+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.trees.TreeNode.$anonfun$transformDownWithPruning$1(TreeNode.scala:461)
[2024-02-27T12:14:23.187+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.trees.CurrentOrigin$.withOrigin(origin.scala:76)
[2024-02-27T12:14:23.187+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.trees.TreeNode.transformDownWithPruning(TreeNode.scala:461)
[2024-02-27T12:14:23.188+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.plans.logical.LogicalPlan.org$apache$spark$sql$catalyst$plans$logical$AnalysisHelper$$super$transformDownWithPruning(LogicalPlan.scala:32)
[2024-02-27T12:14:23.188+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.plans.logical.AnalysisHelper.transformDownWithPruning(AnalysisHelper.scala:267)
[2024-02-27T12:14:23.188+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.plans.logical.AnalysisHelper.transformDownWithPruning$(AnalysisHelper.scala:263)
[2024-02-27T12:14:23.188+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.plans.logical.LogicalPlan.transformDownWithPruning(LogicalPlan.scala:32)
[2024-02-27T12:14:23.188+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.plans.logical.LogicalPlan.transformDownWithPruning(LogicalPlan.scala:32)
[2024-02-27T12:14:23.188+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.trees.TreeNode.transformDown(TreeNode.scala:437)
[2024-02-27T12:14:23.188+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.QueryExecution.eagerlyExecuteCommands(QueryExecution.scala:98)
[2024-02-27T12:14:23.188+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.QueryExecution.commandExecuted$lzycompute(QueryExecution.scala:85)
[2024-02-27T12:14:23.188+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.QueryExecution.commandExecuted(QueryExecution.scala:83)
[2024-02-27T12:14:23.189+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.QueryExecution.assertCommandExecuted(QueryExecution.scala:142)
[2024-02-27T12:14:23.189+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.DataFrameWriter.runCommand(DataFrameWriter.scala:859)
[2024-02-27T12:14:23.189+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.DataFrameWriter.saveToV1Source(DataFrameWriter.scala:388)
[2024-02-27T12:14:23.189+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.DataFrameWriter.saveInternal(DataFrameWriter.scala:361)
[2024-02-27T12:14:23.189+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.DataFrameWriter.save(DataFrameWriter.scala:240)
[2024-02-27T12:14:23.190+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.DataFrameWriter.parquet(DataFrameWriter.scala:792)
[2024-02-27T12:14:23.190+0700] {subprocess.py:93} INFO - 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[2024-02-27T12:14:23.190+0700] {subprocess.py:93} INFO - 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[2024-02-27T12:14:23.190+0700] {subprocess.py:93} INFO - 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[2024-02-27T12:14:23.190+0700] {subprocess.py:93} INFO - 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[2024-02-27T12:14:23.190+0700] {subprocess.py:93} INFO - 	at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:244)
[2024-02-27T12:14:23.190+0700] {subprocess.py:93} INFO - 	at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:374)
[2024-02-27T12:14:23.190+0700] {subprocess.py:93} INFO - 	at py4j.Gateway.invoke(Gateway.java:282)
[2024-02-27T12:14:23.191+0700] {subprocess.py:93} INFO - 	at py4j.commands.AbstractCommand.invokeMethod(AbstractCommand.java:132)
[2024-02-27T12:14:23.191+0700] {subprocess.py:93} INFO - 	at py4j.commands.CallCommand.execute(CallCommand.java:79)
[2024-02-27T12:14:23.191+0700] {subprocess.py:93} INFO - 	at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2024-02-27T12:14:23.191+0700] {subprocess.py:93} INFO - 	at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2024-02-27T12:14:23.191+0700] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:829)
[2024-02-27T12:14:23.191+0700] {subprocess.py:93} INFO - Caused by: java.io.FileNotFoundException: copyFile(olist_order_payments_dataset.parquet/_temporary/0/task_202402271213478392135236523862282_0003_m_000000/part-00000-a686593a-e86f-48d6-9467-e8b024a46016-c000.snappy.parquet, olist_order_payments_dataset.parquet/part-00000-a686593a-e86f-48d6-9467-e8b024a46016-c000.snappy.parquet) on olist_order_payments_dataset.parquet/_temporary/0/task_202402271213478392135236523862282_0003_m_000000/part-00000-a686593a-e86f-48d6-9467-e8b024a46016-c000.snappy.parquet: com.amazonaws.services.s3.model.AmazonS3Exception: Not Found (Service: Amazon S3; Status Code: 404; Error Code: 404 Not Found; Request ID: NNN41GP1M7MJMW0H; S3 Extended Request ID: nUeNRiQgZyOyrOwd6Dp/3G74ywjtdmwXt/O6TCBvdaMRDgxM9fY40421GF+mawJtIHjHgCb0gz0=; Proxy: null), S3 Extended Request ID: nUeNRiQgZyOyrOwd6Dp/3G74ywjtdmwXt/O6TCBvdaMRDgxM9fY40421GF+mawJtIHjHgCb0gz0=:404 Not Found
[2024-02-27T12:14:23.191+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AUtils.translateException(S3AUtils.java:266)
[2024-02-27T12:14:23.191+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.Invoker.once(Invoker.java:119)
[2024-02-27T12:14:23.192+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.copyFile(S3AFileSystem.java:4284)
[2024-02-27T12:14:23.192+0700] {subprocess.py:93} INFO - 	... 65 more
[2024-02-27T12:14:23.192+0700] {subprocess.py:93} INFO - Caused by: com.amazonaws.services.s3.model.AmazonS3Exception: Not Found (Service: Amazon S3; Status Code: 404; Error Code: 404 Not Found; Request ID: NNN41GP1M7MJMW0H; S3 Extended Request ID: nUeNRiQgZyOyrOwd6Dp/3G74ywjtdmwXt/O6TCBvdaMRDgxM9fY40421GF+mawJtIHjHgCb0gz0=; Proxy: null), S3 Extended Request ID: nUeNRiQgZyOyrOwd6Dp/3G74ywjtdmwXt/O6TCBvdaMRDgxM9fY40421GF+mawJtIHjHgCb0gz0=
[2024-02-27T12:14:23.192+0700] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.handleErrorResponse(AmazonHttpClient.java:1819)
[2024-02-27T12:14:23.192+0700] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.handleServiceErrorResponse(AmazonHttpClient.java:1403)
[2024-02-27T12:14:23.192+0700] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeOneRequest(AmazonHttpClient.java:1372)
[2024-02-27T12:14:23.192+0700] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeHelper(AmazonHttpClient.java:1145)
[2024-02-27T12:14:23.192+0700] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.doExecute(AmazonHttpClient.java:802)
[2024-02-27T12:14:23.193+0700] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeWithTimer(AmazonHttpClient.java:770)
[2024-02-27T12:14:23.193+0700] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.execute(AmazonHttpClient.java:744)
[2024-02-27T12:14:23.193+0700] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.access$500(AmazonHttpClient.java:704)
[2024-02-27T12:14:23.193+0700] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutionBuilderImpl.execute(AmazonHttpClient.java:686)
[2024-02-27T12:14:23.193+0700] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient.execute(AmazonHttpClient.java:550)
[2024-02-27T12:14:23.193+0700] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient.execute(AmazonHttpClient.java:530)
[2024-02-27T12:14:23.193+0700] {subprocess.py:93} INFO - 	at com.amazonaws.services.s3.AmazonS3Client.invoke(AmazonS3Client.java:5437)
[2024-02-27T12:14:23.193+0700] {subprocess.py:93} INFO - 	at com.amazonaws.services.s3.AmazonS3Client.invoke(AmazonS3Client.java:5384)
[2024-02-27T12:14:23.193+0700] {subprocess.py:93} INFO - 	at com.amazonaws.services.s3.AmazonS3Client.getObjectMetadata(AmazonS3Client.java:1367)
[2024-02-27T12:14:23.194+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.lambda$getObjectMetadata$10(S3AFileSystem.java:2545)
[2024-02-27T12:14:23.194+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.Invoker.retryUntranslated(Invoker.java:414)
[2024-02-27T12:14:23.194+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.Invoker.retryUntranslated(Invoker.java:377)
[2024-02-27T12:14:23.194+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.getObjectMetadata(S3AFileSystem.java:2533)
[2024-02-27T12:14:23.194+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.lambda$copyFile$28(S3AFileSystem.java:4286)
[2024-02-27T12:14:23.194+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.Invoker.once(Invoker.java:117)
[2024-02-27T12:14:23.194+0700] {subprocess.py:93} INFO - 	... 66 more
[2024-02-27T12:14:23.194+0700] {subprocess.py:93} INFO - 
[2024-02-27T12:14:23.917+0700] {subprocess.py:97} INFO - Command exited with return code 1
[2024-02-27T12:14:24.202+0700] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/nhut/etl-airflow/.venv/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/nhut/etl-airflow/.venv/lib/python3.10/site-packages/airflow/operators/bash.py", line 212, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2024-02-27T12:14:24.246+0700] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=missed_deadline_job_dag, task_id=spark_ingestion_to_s3, execution_date=20240226T092846, start_date=20240227T050712, end_date=20240227T051424
[2024-02-27T12:14:24.409+0700] {standard_task_runner.py:107} ERROR - Failed to execute job 153 for task spark_ingestion_to_s3 (Bash command failed. The command returned a non-zero exit code 1.; 32232)
[2024-02-27T12:14:24.462+0700] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-02-27T12:14:24.705+0700] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
